### ISSUE TITLE
Building Blocks (VIPER) CD work for DMZ environment deployment

### ISSUE THREAD

**Issue by rin-skylight** [CDCgov/dibbs-cloud#55]
### Background
The VIPER building block set needs to deploy to the DMZ environment for CDC programs as well as the usual one. We need to work with the team to ensure that is possible.

### Action requested
Add workflows for gated production to DMZ-stg and DMZ-prod.

### Acceptance Criteria
- Workflow changes for DMZ-stg exist
- Workflow changes for DMZ-prod exist
- Deployments are gated on successful deployment to lower environment
- Deployments are successful
- Failed deployment to stg level blocks deployment to prod level without affecting other environment path.

**Comment by rin-skylight** [comment]
Architecturally obsolete. Closing as "Won't Do".
